devscripts/bash-completion.in

```diff
@@ -10,9 +10,13 @@ __yt_dlp()
     diropts="--cache-dir"
 
     if [[ ${prev} =~ ${fileopts} ]]; then
+        local IFS=$'\n'
+        compopt -o filenames
```

Comment:
The `compopt` builtin was added in bash 4.0 per https://mywiki.wooledge.org/BashFAQ/061

macOS still ships with bash 3.2. while newer macOS versions also ship with zsh and make that the default shell, older macOS systems will still be running ancient bash by default.

we need to check if `compopt` is available and only then use it
```suggestion
        type compopt > /dev/null 2>&1 && compopt -o filenames
```

Reply:
Thanks! I don't have an old bash version to check, but I amended this and checked with 5.2. Perhaps someone else can test with 3.x.

I also chose to use the `&>` bashism for redirection.